packages/common/src/system/guid.ts

```diff
@@ -6,13 +6,13 @@ import * as uuid from 'uuid-with-v6';
 // tslint:disable: no-unsafe-any
 
 export namespace Guid {
-    export function createGuid(lastGuid?: string): string {
-        if (lastGuid === undefined) {
-            return uuid.v6();
-        }
+    export function createGuid(): string {
+        return uuid.v6();
+    }
 
+    export function createGuidForNode(baseGuid: string): string {
```

Comment:
what does forNode mean? may be rename to createFromGuid?

Reply:
https://tools.ietf.org/html/rfc4122#section-4.1.6

Reply:
may be better to add this link here so that it is clear what node means.

Reply:
I did add doc comments to all guid functions w/ explanation - is this not clear enough? U can refer to RFC 4122 as well - link on top of the file